tests/features/drush.feature

```diff
@@ -1,4 +0,0 @@
-Feature: drush
```

Comment:
I added a test for an unavailable Drush command and for an unavailable WP command.